tests/unit/functions/server_functions_spec.js

```diff
@@ -0,0 +1,136 @@
+require("isomorphic-fetch");
```

Comment:
Maybe it's obvious, but I don't see it. What can `isomorphic-fetch` do that the current implementation of `fetch` cannot?

Reply:
It won't be in the final version 👍 

It was an attempt to fix an error that occurred when I pushed to GitHub. Suddenly I got an [error](https://github.com/MichMich/MagicMirror/actions/runs/3322851239) saying the `Response` is not defined. I got [the idea to test it from StackOverflow](https://stackoverflow.com/questions/52420318/referenceerror-response-is-not-defined). The real reason turns out to be different node versions on GitHub and what I use locally, so [GitHub uses `node-fetch` instead of `global.fetch`](https://github.com/MichMich/MagicMirror/blob/9e0293047f8a8c0ced7d9fd8de1b839c06baf9cb/js/fetch.js).

Reply:
Removed all references to `isomorphic-fetch` now.